operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -400,20 +418,31 @@ Please refer [iSCSI Metrics](https://github.com/Cray-HPE/sbps-marshal/blob/main/
 
 ## Steps to switch from DVS to iSCSI
 
+This involves this sequence of operations:
+
 * Disable DVS
 * Uninstall CPS
 * Enable iSCSI SBPS
 
-To disable DVS please refer:
-<https://github.hpe.com/hpe/hpc-shasta-os-uss-docs/blob/master/dac-assets/documents/operations/DVS_and_SBPS.md>
+To perform these operations, please refer the section `Switch from DVS to iSCSI` documented in the publication:
```

Comment:
The name of the section is "DVS and SBPS", not "Switch from DVS to iSCSI".

Reply:
ok, will incorporate the same. Thanks.